src/modules/github/components/pull-review-form/pull-review-form.tsx

```diff
@@ -28,6 +30,7 @@ interface Errors {
 export const PullReviewForm: FC<PullReviewFormProps> = ({
   repoHasCheckArray
 }) => {
+  const [currentUser, setCurrentUser] = useState<string>()
```

Comment:
`const [currentUser, setCurrentUser] = useState<string>('')`
You forgot to initialize useState